2.0/TransmissionTest/glTF/TransmissionTest.gltf

```diff
@@ -0,0 +1,1522 @@
+            "target": 34962
+        },
+        {
+            "buffer": 0,
+            "byteOffset": 204768,
+            "byteLength": 64,
+            "target": 34962
+        },
+        {
+            "buffer": 0,
+            "byteOffset": 204832,
+            "byteLength": 96,
+            "target": 34962
+        },
+        {
+            "buffer": 0,
+            "byteOffset": 204928,
+            "byteLength": 96,
+            "target": 34962
+        },
+        {
+            "buffer": 0,
+            "byteOffset": 205024,
+            "byteLength": 64,
+            "target": 34962
+        },
+        {
+            "buffer": 0,
+            "byteOffset": 205088,
+            "byteLength": 96,
+            "target": 34962
+        },
+        {
+            "buffer": 0,
+            "byteOffset": 205184,
+            "byteLength": 96,
+            "target": 34962
+        },
+        {
+            "buffer": 0,
+            "byteOffset": 205280,
+            "byteLength": 64,
+            "target": 34962
+        }
+    ],
+    "buffers": [
+        {
+            "byteLength": 205344,
+            "uri": "TransmissionTest_binary.bin"
+        }
+    ],
+    "cameras": [
+        {
+            "perspective": {
+                "znear": 0.00005606882768915966,
+                "yfov": 0.3024156808853149,
+                "zfar": 3.0837855339050295,
+                "aspectRatio": 1.3333333730697632
+            },
+            "type": "perspective",
+            "name": "render_camera"
+        }
+    ],
+    "images": [
+        {
+            "name": "texture14184",
+            "uri": "TransmissionTest_images/texture14184.jpg",
+            "mimeType": "image/jpeg"
+        },
+        {
+            "name": "texture214190",
+            "uri": "TransmissionTest_images/texture214190.png",
+            "mimeType": "image/png"
+        },
+        {
+            "name": "texture4086",
+            "uri": "TransmissionTest_images/texture4086.png",
+            "mimeType": "image/png"
+        },
+        {
+            "name": "texture177328",
+            "uri": "TransmissionTest_images/texture177328.png",
+            "mimeType": "image/png"
+        },
+        {
+            "name": "texture6807",
+            "uri": "TransmissionTest_images/texture6807.png",
+            "mimeType": "image/png"
+        },
+        {
+            "name": "texture175763",
+            "uri": "TransmissionTest_images/texture175763.png",
+            "mimeType": "image/png"
+        },
+        {
+            "name": "texture10487",
+            "uri": "TransmissionTest_images/texture10487.png",
+            "mimeType": "image/png"
+        },
+        {
+            "name": "texture927751",
+            "uri": "TransmissionTest_images/texture927751.jpg",
+            "mimeType": "image/jpeg"
+        },
+        {
+            "name": "texture927748",
+            "uri": "TransmissionTest_images/texture927748.jpg",
```

Comment:
This image is flagged by the glTF Validator as not being used, and not being present on disk.

It should have failed CI with this, but it looks like CI isn't running.

Reply:
Thank you. I was editing it by hand before submitting and wasn't careful enough.